ogcapi-features/README.md

```diff
@@ -1,42 +1,93 @@
-# STAC API - Features
-
-- [STAC API - Features](#stac-api---features)
+# STAC API - Features and Collections Specification
+
+- [STAC API - Features and Collections Specification](#stac-api---features-and-collections-specification)
+  - [Conformance Classes](#conformance-classes)
+    - [STAC API - Features](#stac-api---features)
```

Comment:
Doesn't this actually require STAC API - Core and thus implicitly require all the OGC API classes? Wondering whether we should just list STAC API - Core here and not require the OGC classes explicitly, otherwise we'd need to do that basically everywhere?!

Reply:
I think we need to decide if this section means all of the dependent conformance classes also, or only the ones defined in this spec. I think it should  be just what's defined in this spec, since core is the only one that's required by any api.

Core doesn't implicitly **require** the OGC API classes -- those are optional if you want to advertise that this is also an OGC API, but you don't have to.

Reply:
Yes, I agree that should be just what's defined in this spec. But then this doesn't need to list GeoJSON or OpenAPI, right?

Reply:
Agree, I'll remove those here and clarify the text

Reply:
clarified.